YearInReview/Model/Reports/ReportGenerator.cs

```diff
@@ -45,6 +47,7 @@ public async Task<IReadOnlyCollection<Report1970>> GenerateAllYears()
 				var filteredActivities = _specificYearActivityFilter.GetActivityForYear(year, activities);
 				var report = _composer1970.Compose(year, filteredActivities);
 				reports.Add(report);
+				ShowReportGeneratedNotification(year);
```

Comment:
Good idea about showing a single notification for this initial generation. No need to span user. So we can either have a notification with custom text (without specifying year), or we could make sure that the notification is shown for the most recent report.